.github/workflows/rds-plan.yml

```diff
@@ -0,0 +1,40 @@
+name: rds plan terraform
+
+on:
+  pull_request:
+    paths:
+      - terraform/services/tfstate/rds/**
+  workflow_dispatch: # Allow manual trigger
+
+jobs:
+  terraform-plan:
+    needs: check-terraform-fmt
+    permissions:
+      contents: read
+      id-token: write
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./terraform/services/tfstate
+    strategy:
+      fail-fast: false
+      matrix:
+        app: [ab2d, bcda, dpc]
+        env: [dev, test, sbx, prod]
+        include:
+          - app: ab2d
+            env: mgmt
+          - app: bcda
+            env: mgmt
```

Comment:
The mgmt environments should be dropped as there are no API RDS databases in those environments.